app/views/submissions/show.erb

```diff
@@ -43,7 +43,7 @@
 
     <div class="clearfix"></div>
     <div class="submission-author text-muted">
-      submitted by <%= gravatar_tag submission.user.avatar_url, size: 20 %>
+      submitted in <%= Trackler.tracks[submission.language].language %> by <%= gravatar_tag submission.user.avatar_url, size: 20 %>
```

Comment:
"Minor unimportant, feel free to ignore" feedback:
What do you think about putting each template replacement on it's own line to match line 47 and 48?
```
submitted in <%= Trackler.tracks[submission.language].language %>
by <%= gravatar_tag submission.user.avatar_url, size: 20 %>       
<%= profile_link(submission.user) %>
<%= ago(submission.created_at) %>

```